### ISSUE TITLE
swagger giving `Cannot GET /index.html`

### ISSUE THREAD

**Issue by solerf** [gofiber/swagger#100]
I'm adding swagger in an application following the `README.md` in the repository home, as it is.

My problem is when accessing `http://localhost:8080/swagger` the url is rewritten to `http://localhost:8080/index.html` and I get the message from the title. I don't get why it is being rewritten.

The swagger endpoint handler is configured as in the `README.md`:
```go
app := fiber.New()

app.Get("/swagger/*", swagger.HandlerDefault)
// other routes
```
and dependencies version:
```go
github.com/gofiber/fiber/v2 v2.52.5
github.com/gofiber/swagger v1.1.0
```

Any ideas?

**Comment by borislavpanov-web** [comment]
I ran into the same issue where http://localhost:8080/swagger was redirecting to http://localhost:8080/index.html. After some digging, I found out that the problem was due to cached data in my browser.

What worked for me:
- I cleared the cookies and site data for localhost in the browser settings.
- Once I did that and refreshed the page, the Swagger UI loaded just fine without any redirects.
- If that doesn’t work for you, try clearing all browsing data (cache, cookies, etc.), as there might be something else interfering.
